### DISCUSSION TITLE
Export multiple markdowns to multiple epub

### DISCUSSION THREAD

**Discussion by Cronos-lab** [jgm/pandoc#9491] [Q&A]
Hello,
I'd like to export several markdowns into several epub files in the same folder. But also with the same name as the original file.

If it were possible to export the files in a single command, that would be great.

I would like the answer for Linux and Windows.

I looked for an answer but couldn't find it.

Thank you so much.

**Comment by iandol** [comment]
Use something like pandocomatic:

https://github.com/htdebeer/pandocomatic

Or write a script with a loop. Pandoc is strictly one document in one document out as far as I undersand...

**Reply by Cronos-lab** [reply]
Thank you !

Unfortunately, I don't have enough computer knowledge to configure all that.

**Reply by jgm** [reply]
See
https://pandoc.org/faqs#how-can-i-convert-a-whole-directory-of-files-from-markdown-to-rtf

**Comment by Cronos-lab** [comment]
Thank you.

I managed to do it with pandocomatic.
